jax/experimental/jax2tf/examples/tf_js/quickdraw/README.md

```diff
@@ -0,0 +1,38 @@
+# Quickdraw
+
+This directory contains the code necessary to train a CNN on the Quickdraw
+dataset using Flax, and subsequently convert it using jax2tf into a model that
+can be loaded in TF.js.
+
+## Training the model
+
+To train and export the model yourself, simply run
```

Comment:
Is this step going to take a while? It would be good to tell the user, something like "You can train and export the model yourself, which will take 30 minutes, or you can skip to [next section](#interacting-with-the-model)"